### PR TITLE
Add Zuku271's Warsaw file

### LINKED ISSUE #33: Zuku271 Workflow for adding Warsaw to the Repository
## Warsaw

```
- [x] Create an issue
- [ ] Create a branch
- [ ] Make changes
- [ ] Open a PR
- [ ] Collaborate/Have a discussion
- [ ] Get approval
- [ ] Merge
```

### PR DESCRIPTION
**Pull request by adrianz261**
closes #33

### CODE REVIEW

File: Zuku271-Warsaw.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Warsaw
+
+### Restaurants
+
```
Comment (sskhub) [inline Zuku271-Warsaw.md:4]: ### Good Job :+1:

Comment (ghost) [inline Zuku271-Warsaw.md:4]: Where are the best places to go for a meal?

File: Zuku271-Warsaw.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## Warsaw
+
```
Comment (ghost) [inline Zuku271-Warsaw.md:2]: What are the most interesting places to go and see in Warsaw?

**Review by sskhub — APPROVED** [review]

**Comment by aarboleda1** [comment]
nice work ! 👍

File: Zuku271-Warsaw.md (hunk 2 above)
Comment (jaimemendozadev) [reply Zuku271-Warsaw.md:2]: How about a museum? I love going to museums.

**Pull request merged**